Documentation/project-docs/support-dotnet-core-instructions.md

```diff
@@ -22,11 +22,11 @@ After running the analysis, you will see your .NET Portability Report. Only type
 
 ![](../images/portability_report.png)
 
-Don't want to use Visual Studio? You can also use the Portability Analyzer from the Command Prompt. Download the command-line analyzer [here](http://www.microsoft.com/en-us/download/details.aspx?id=42678).
+Don't want to use Visual Studio? You can also use the Portability Analyzer from the Command Prompt. Download the command-line analyzer [here](http://github.com/microsoft/dotnet-apiport/releases).
 
-- Type the following command to analyze the current directory: ```\...\ApiPort.exe .  ```
-- To analyze a specific list of .dlls type the following command: ```\...\ApiPort.exe first.dll second.dll third.dll ```
+- Type the following command to analyze the current directory: `ApiPort.exe analyze -f . `
+- To analyze a specific list of .dlls type the following command: `ApiPort.exe analyze -f first.dll -f second.dll -f third.dll`
```

Comment:
It should work on Linux/OSX under Mono, but I haven't tried it for a while. It fails on Windows under Mono, but for a Windows specific issue.